concepticondata/conceptlists/Luangthongkum-2019-341.tsv

```diff
@@ -106,12 +106,12 @@ Luangthongkum-2019-341-104	104	enter	749	ENTER
 Luangthongkum-2019-341-105	105	exert forcing to expel		
 Luangthongkum-2019-341-106	106	extinguish (t.v.)	127	EXTINGUISH
 Luangthongkum-2019-341-107	107	eye, face		
-Luangthongkum-2019-341-108	108	fall down	1280	FALL
+Luangthongkum-2019-341-108	108	fall down	2894	TUMBLE (FALL DOWN)
```

Comment:
most previous cases of "fall down" are mapped to FALL, so I would stick with this